### PR TITLE
Update the linux kernel versions in boot and npb tutorials

### PR DESCRIPTION
**Pull request by aakahlow**
These are changes in boot and npb tutorials and launch scripts to use latest linux kernel and last four LTS releases.

### CODE REVIEW

File: docs/boot-tutorial.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ linux_binaries = {
                 cp ../linux-configs/config.{version} .config;
                 make -j8;
                 cp vmlinux vmlinux-{version};
-                '''.format(v='5.2.3'),
+                ''',
                 inputs = [experiments_repo, linux_repo,],
                 documentation = f"Kernel binary for {version} with simple "
```
Comment (powerjg) [inline docs/boot-tutorial.md:278]: I think this needs a `.format`

File: docs/launch_boot_tests.py [hunk 2]
```diff
@@ -90,7 +90,7 @@
                 cp ../linux-configs/config.{version} .config;
                 make -j8;
                 cp vmlinux vmlinux-{version};
-                '''.format(v='5.2.3'),
+                ''',
                 inputs = [experiments_repo, linux_repo,],
                 documentation = f"Kernel binary for {version} with simple "
```
Comment (powerjg) [inline docs/launch_boot_tests.py:95]: `.format`

**Review by powerjg — COMMENTED** [review]
One very small comment.

File: docs/boot-tutorial.md (hunk 1 above)
Comment (aakahlow) [reply docs/boot-tutorial.md:278]: Is .format really needed? These are f strings and {version} is defined by the outer loop (which is iterating through a list of different linux versions)

**Comment by powerjg** [comment]
I see. There's a mix of .format and f strings. We should pick one or the
other.

Since we're using python3 and require at least 3.5 for gem5art, I would
vote for using f strings everywhere.

Jason

**Review by powerjg — APPROVED** [review]
Push whenever you feel it's ready!

**Comment by aakahlow** [comment]
> I see. There's a mix of .format and f strings. We should pick one or the other. Since we're using python3 and require at least 3.5 for gem5art, I would vote for using f strings everywhere. Jason

Okay!

**Pull request merged**